frontend/components/FieldDateTime/FieldDateTimeEdit.jsx

```diff
@@ -201,6 +201,12 @@ export default class FieldDateTimeEdit extends Component {
 
   handleFocus(hasFocus) {
     const {pickerVisible} = this.state
+    const {schema} = this.props
+
+    // Return from focus event so picker doesn't display
+    if (schema.publish.readonly === true) {
```

Comment:
Is it worth adding a check for `schema.publish`, in case it doesn't exist? I don't think we are ensuring that property exists anywhere upstream.